authz/audit/audit_logging_test.go

```diff
@@ -271,7 +271,8 @@ func (s) TestAuditLogger(t *testing.T) {
 				grpc.ChainUnaryInterceptor(i.UnaryInterceptor),
 				grpc.ChainStreamInterceptor(i.StreamInterceptor))
 			defer s.Stop()
-			testgrpc.RegisterTestServiceServer(s, ss)
+			ss.S = s
+			stubserver.StartTestService(t, ss)
```

Comment:
we have to remove the redundant server below?

Reply:
yeah, removed redundant code